agent/docker/default_config.yaml

```diff
@@ -6,10 +6,9 @@ orb:
         token_url: ${FLEET_AUTH_URL}
         client_id: ${FLEET_CLIENT_ID}
         client_secret: ${FLEET_CLIENT_SECRET}
+        otlp_bridge_grpc_port: 4337
```

Comment:
add a port to ensure that agent will not fail if there is a otel collector already running